<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="DESKTOP-9VAR55H" name="com.testPackage_1.TestClass_1_P1" tests="5" failures="0" timestamp="24 Mar 2021 06:57:48 GMT" time="0.031" errors="1">
  <testcase name="view" time="0.000" classname="com.testPackage_1.TestClass_1_P1">
    <skipped/>
  </testcase> <!-- view -->
  <testcase name="show" time="0.031" classname="com.testPackage_1.TestClass_1_P1">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.testPackage_1.TestClass_1_P1.show() didn&#039;t finish within the time-out 2">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.testPackage_1.TestClass_1_P1.show() didn't finish within the time-out 2
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- show -->
  <testcase name="Display" time="0.000" classname="com.testPackage_1.TestClass_1_P1"/>
  <testcase name="Display" time="0.000" classname="com.testPackage_1.TestClass_1_P1"/>
  <testcase name="Display" time="0.000" classname="com.testPackage_1.TestClass_1_P1"/>
</testsuite> <!-- com.testPackage_1.TestClass_1_P1 -->
